modules/bitrate_controller/send_side_bandwidth_estimation.cc

```diff
@@ -422,7 +422,10 @@ void SendSideBandwidthEstimation::CapBitrateToThresholds(int64_t now_ms,
   if (bwe_incoming_ > 0 && bitrate_bps > bwe_incoming_) {
     bitrate_bps = bwe_incoming_;
   }
-  if (delay_based_bitrate_bps_ > 0 && bitrate_bps > delay_based_bitrate_bps_) {
+  if (delay_weight == 100) {
+    bitrate_bps = delay_based_bitrate_bps_;
+  }
+  else if (delay_based_bitrate_bps_ > 0 && bitrate_bps > delay_based_bitrate_bps_) {
```

Comment:
format the code to put else if right after { above.

Reply:
I think it's already formatted the way you're requesting:

https://github.com/open-webrtc-toolkit/owt-deps-webrtc/blob/db0c98321646b0463e809423499f547be0eb1f62/modules/bitrate_controller/send_side_bandwidth_estimation.cc#L422-L430

Reply:
Oh I see what you mean.